docs/user/getting-started.md

```diff
@@ -22,16 +22,16 @@ For detailed information on installing, configuring and upgrading Kubeapps, chec
 
 The above commands will deploy Kubeapps into the `kubeapps` namespace in your cluster. It may take a few minutes to execute. Once it has been deployed and the Kubeapps pods are running, continue to step 2.
 
-## Step 2: Create a Kubernetes API token
+## Step 2: Create a demo credential with which to access Kubeapps and Kubernetes
 
-For trying out Kubeapps, access to the Dashboard requires a Kubernetes API token to authenticate with the Kubernetes API server as shown below, but for any real installation of Kubeapps you should instead [configure an OAuth2/OIDC provider](using-an-OIDC-provider.md).
+For any user-facing installation you should [configure an OAuth2/OIDC provider](using-an-OIDC-provider.md) to enable secure user authentication with Kubeapps and the cluster, but this is quite an overhead to simply try out Kubeapps. Instead, for a simpler way to try out Kubeapps in a non-user-facing environment, we can create a Kubernetes service account and use that API token to authenticate with the Kubernetes API server via Kubeapps:
```

Comment:
Suggestion:
`Instead, for a simpler way to try out Kubeapps in a non-user-facing environment, we can create a Kubernetes service account` -->  `Instead, for a simpler way to try out Kubeapps, we can create a Kubernetes service account` 
(the user-facing term is a bit confusing, at least for me, to repeat it)

Reply:
Yeah, I just want to be clear that it's not a simple way for trying out and continue to use Kubeapps. Perhaps, `Instead, for a simpler way to try out Kubeapps for personal learning or demoing, we can ...`? I'll update to that